2020/cfp.md

```diff
@@ -0,0 +1,57 @@
+                   Call for Participation: VMM'2020
+                   ================================
+
+                      6th Virtual Machine Meetup
+
+                         Co-located with MPLR
+                        as virtual online event
+                  November 4-6th, 2020, Manchester, UK
+                   
+                    http://vmmeetup.github.io/2020/
+
+
+The 6th Virtual Machine Meetup (VMM'2020) is a venue for discussing the latest
+research and developments in the area of managed language execution. It will be
+held on November 4-6th as a virtual event, and is colocated with the Managed Programming
+Languages & Runtimes (MPLR'20) conference (https://mplr2020.cs.manchester.ac.uk/).
+We welcome presentations of new research results, experience reports, as well as
+position statements that can lead to interesting discussions.
+
+
+Topics include, but are not limited to:
+
+ - Programming language design
+ - Dynamic and static program analysis
+ - Compiler construction
+ - Managed runtime architectures
+ - Data processing engines
+ - Distributed execution environments
```

Comment:
Id add graalvm focus stuff as well like "interesting developments in the GraalVM eco system"

Reply:
I've put ` - Developments in the GraalVM ecosystem and related topics, including` as first bullet point.